server/base/src/main/java/org/apache/accumulo/server/master/balancer/RegexGroupBalancer.java

```diff
@@ -68,7 +69,8 @@ protected long getWaitTime() {
   @Override
   protected Function<KeyExtent,String> getPartitioner() {
 
-    Map<String,String> customProps = configuration.getTableConfiguration(tableId).getAllPropertiesWithPrefix(Property.TABLE_ARBITRARY_PROP_PREFIX);
+    Map<String,String> customProps = tabletBalancerEnvironment.getConfiguration().getTableConfiguration(tableId)
```

Comment:
Would we also want to abstract out all places where we access tableConfiguration? Replacing them with a propery map? Like the ones below in TableLoadBalancer (Property.TABLE_CLASSPATH, Property.TABLE_LOAD_BALANCER)